src/main/java/com/iexec/common/security/CipherUtils.java

```diff
@@ -125,10 +128,12 @@ public static byte[] stripIvFromEncryptedData(byte[] dataWithIv) {
      * Concatenate IV (initialization vector) to the
      * beginning of the encrypted data.
      *
-     * @param iv
-     * @param encryptedData
-     * @return
-     * @throws IOException
+     * @param iv            Initialization vector
+     * @param encryptedData Encrypted data
+     * @return IV prepended to the encrypted data
+     * as an array of bytes.
```

Comment:
For small sentences like this, I think it could be more readable to write on a single line
```suggestion
     * @return IV prepended to the encrypted data as an array of bytes.
```
This way, each line will correspond to a single javadoc statement

Reply:
Resolved in [cc09d9b](https://github.com/iExecBlockchainComputing/iexec-common/pull/470/commits/cc09d9b58dd26d71e3c901ac02505ef4264cb657)

Reply:
Resolved in [cc09d9b](https://github.com/iExecBlockchainComputing/iexec-common/pull/470/commits/cc09d9b58dd26d71e3c901ac02505ef4264cb657)